pkgs/by-name/ni/nist-feed/package.nix

```
@@ -0,0 +1,50 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, makeWrapper
+, bash
+, jq
+, killall
+, libnotify
+, cron
+}:
+
+stdenv.mkDerivation rec {
```

`rec` doesn't seem to be used

```suggestion
stdenv.mkDerivation {
```